spec/all.globalBeforeEach.spec.js

```diff
@@ -0,0 +1,91 @@
+  describe('Testing - CFTO UK Site', function() {
+
+
+function log_in()
+{
+  browser.setValue('#loginEmail', 'sroy@sapient.com');
+  browser.setValue('#loginPassword', 'welcome1');
+}
+
+function isMerge()
+{
+  var el = browser.getTitle();
+  if(el == 'My Basket')
+  {
+  browser.click('button[data-analyticsfunction="analyticsCheckoutButtonClicked"]');
+  browser.setValue('.collection--store-finder__search-form__input', 'Covent Garden');
+  }
+  else {
+  browser.setValue('.collection--store-finder__search-form__input', 'Covent Garden');
+  }
+}
+
+
+it.only('Place an UK CFTO order with registered user with added item', function() {
+
+    this.timeout(0);
+    browser.click('.add-to-bag__btn');
+    browser.waitForVisible('script + .minibag-container', 10000);
+    browser.click('script + .minibag-container .minibag-summary__button');
+    browser.click('button[data-analyticsfunction="analyticsCheckoutButtonClicked"]');
+    log_in();
+    browser.waitForEnabled('.my-account__subcontent__signIn-btn', 10000);
+    browser.click('.my-account__subcontent__signIn-btn');
+    isMerge();
+    browser.click('.collection--store-finder__search-form__btn');
+    browser.click('.btn--primary');
+    browser.click('.btn--primary');
```

Comment:
I have commented all the functions & steps and transformed the redundant code into functions